### PR TITLE
(SDI-2095) Add latest tag to builds

### PR DESCRIPTION
**Pull request by nanliu**
- add version builds
- add latest tag to xenial

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -21,10 +21,14 @@ rescue LoadError
...
 
-def build(os, org="intelsdi")
+def build(os, org="intelsdi", version=nil)
   raise "Error: missing operating system" unless os
   raise "Error: invalid operating system #{os}" unless File.exists? "#{os}/Dockerfile"
-  sh "docker build -t #{org}/snap:#{os} --build-arg BUILD_DATE=$(date +%Y-%m-%d) -f #{os}/Dockerfile ."
+  if version
+    sh "docker build -t #{org}/snap:#{version}_#{os} --build-arg BUILD_DATE=$(date +%Y-%m-%d) --build-arg SNAP_VERSION=#{version} -f #{os}/Dockerfile ."
+  else
+    sh "docker build -t #{org}/snap:#{os} --build-arg BUILD_DATE=$(date +%Y-%m-%d) -f #{os}/Dockerfile ."
```
Comment (candysmurf) [inline Rakefile]: if version=nil, do you assume the lastest version?

File: docker_push.sh [hunk 2]
```diff
@@ -5,4 +5,4 @@ set -u
 set -o pipefail
 
 
-find . -not -path '*/\.*' -name Dockerfile -exec dirname {} \; | cut -c 3- | xargs -n1 -I{} docker push instelsdi/snap:{}
+find . -not -path '*/\.*' -name Dockerfile -exec dirname {} \; | cut -c 3- | xargs -n1 -I{} docker push intelsdi/snap:{}
```
Comment (candysmurf) [inline docker_push.sh]: no version and platform?

File: Rakefile (hunk 1 above)
Comment (nanliu) [inline Rakefile]: It depends on the container Dockerfile, but yes in general latest:
https://github.com/intelsdi-x/snap-docker/blob/master/alpine/Dockerfile#L5

**Comment by candysmurf** [comment]
LGTM

**Pull request merged**